### PR TITLE
Update build env for newer version of mdiswiz

### LINKED ISSUE #256: MDIS Wizzard build environment / migration to newer Qt
We need to put together a new build environment so that we can generate the MDIS wizzard binaries as they are today. 

The current ones in the repo seem to have pretty old libraries that are statically linked. This is so because they presumably were built inside a very old Linux distribution (ubuntu 10, SUSE Linux Enterprise Server 10 or so). 

The major issue with the old static libraries seems to be with the libX and related libraries when using QT version 4.6.0. The application crashes at runtime if not linked against an old enough libX library. 

```
[xcb] Unknown sequence number while appending request
	[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
	[xcb] Aborting, sorry about that.
	mdiswiz_vagrant: ../../src/xcb_io.c:157: append_pending_request: Assertion `!xcb_xlib_unknown_seq_number' failed.
```

We have been able to link against the X libraries from a backup which seems to suggest it was X version 1.6.2. 

```
-rw-r--r-- 1 vagrant vagrant 1592854 Oct  4  2007 /home/vagrant/13Y008-90/NATIVE/DIST_GCC3/usr/X11R6/lib/libX11.a
lrwxrwxrwx 1 vagrant vagrant      13 May 12  2020 /home/vagrant/13Y008-90/NATIVE/DIST_GCC3/usr/X11R6/lib/libX11.so -> libX11.so.6.2
lrwxrwxrwx 1 vagrant vagrant      13 May 12  2020 /home/vagrant/13Y008-90/NATIVE/DIST_GCC3/usr/X11R6/lib/libX11.so.6 -> libX11.so.6.2
-rwxr-xr-x 1 vagrant vagrant 1029416 May 18  2007 /home/vagrant/13Y008-90/NATIVE/DIST_GCC3/usr/X11R6/lib/libX11.so.6.2
```

### PR DESCRIPTION
**Pull request by mad-jsanjuan**
These patches are so that we can build mdiswiz binaries using a backup of the old buildserver as sysroot. This will let us generate binaries that are as close as possible to what has been generated in the past. 

For further updates and new developments we should consider migrating the codebase to a newer Qt version.

### CODE REVIEW

**Review by mad-jrodriguez — DISMISSED** [review]
The changes look good to me

File: README.md [hunk 1]
```diff
@@ -8,14 +8,20 @@ The MDIS wizard contains all CPUs and carrier boards for configuring a system.
...
 
-to build the MDIS Wizard the repo must be checked out on the old software server (192.1.1.22) because
-of QT and tool chain available only there.
+The MDIS Wizzard`s code is based on fairly old QT4 code with QT3 support in it. To build it a compatible
+compilation sysroot is necessary. This sysroot comes from an old software server at MeN (used to
+be at 192.1.1.22) and it contains a Qt 4.6.0 compilation in it and all the static libraries that are
+needed to create the binary (X11, fontconfig, etc.).
+
+The location of these sysroot needs to be defined in the `builddist.env` file. The variable `QTSYSROOT`
+tells the Makefile where to find all the necessary dependencies and tools.
```
Comment (mad-jrodriguez) [inline README.md:17]: Additionally, we should tell in the README that the package gcc-multilib should be installed for 32bit support

File: NATIVE/DIST_GCC3/Makefile [hunk 2]
```diff
@@ -2039,11 +2005,5 @@ compiler_clean: compiler_moc_header_clean
...
-####### Install
-
-install:   FORCE
-
-uninstall:   FORCE
-
-FORCE:
-
+release: all
+	$(STRIP) $(INSTALL_ROOT)/home/jorge/Work/MDIS/13Y008-90/NATIVE/DIST_GCC3/mdiswiz
```
Comment (mad-jrodriguez) [inline NATIVE/DIST_GCC3/Makefile]: This path is incorrect ans should replaced with:
```
$(STRIP) mdiswiz
```

File: README.md (hunk 1 above)
Comment (mad-jsanjuan) [reply README.md:17]: We should not need any package and everything should come from the sysroot. @mad-jrodriguez  and myself have found and issue where actually it was a missing -I option for the compiler so everything is found in the compilation sysroot.

**Review by mad-jrodriguez — APPROVED** [review]
Looks good to me.

File: README.md (hunk 1 above)
Comment (mad-jrodriguez) [reply README.md:17]: Yes, you are right. Now, it works properly, only adding -I$(QTSYSROOT)/usr/include

**Pull request merged**